email_processing/ticket_description_generation/test_data/populate.py

```diff
@@ -0,0 +1,33 @@
+import csv
+from dataclasses import dataclass
+from typing import List
+import os, sys, email
+import numpy as np
+import pandas as pd
+
+@dataclass
+class Email:
+    to: List[str]
+    author: str
+    date: str
+    subject: str
+    body: str
+
+def populate_class(filename):
+    df = pd.read_csv(filename, sep='\t')
+    emails = []
+    #'To', 'From', 'Subject', 'X-To', 'X-From', 'content'
+    for index, row in df.iterrows():
+        to = row['to']
+        author = row['author']
+        date = row['date']
+        subject = row['subject']
+        body = row['body']
+
+        email = Email(to, author, date, subject, body)
+        emails.append(email)
+
+    return emails
+
+if __name__ == '__main__':
+    populate_class('your_file.csv')
```

Comment:
When testing the file on the csv file provided I am getting this issue. It could just be a local issue on my end with something unconfigured. I just want to make sure it doesnt have to do with the file itself:
```
populate_class('task_emails.csv')
  File "C:\Users\Lessa\OneDrive\Documents\UOFT\blueprint\tfss\email_processing\ticket_description_generation\test_data\populate.py", line 17, in populate_class
    df = pd.read_csv(filename, sep='\t')
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lessa\AppData\Local\Programs\Python\Python311\Lib\site-packages\pandas\io\parsers\readers.py", line 912, in read_csv
    return _read(filepath_or_buffer, kwds)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lessa\AppData\Local\Programs\Python\Python311\Lib\site-packages\pandas\io\parsers\readers.py", line 583, in _read
    return parser.read(nrows)
           ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lessa\AppData\Local\Programs\Python\Python311\Lib\site-packages\pandas\io\parsers\readers.py", line 1704, in read
    ) = self._engine.read(  # type: ignore[attr-defined]
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Lessa\AppData\Local\Programs\Python\Python311\Lib\site-packages\pandas\io\parsers\c_parser_wrapper.py", line 234, in read
    chunks = self._reader.read_low_memory(nrows)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "pandas\_libs\parsers.pyx", line 812, in pandas._libs.parsers.TextReader.read_low_memory
  File "pandas\_libs\parsers.pyx", line 873, in pandas._libs.parsers.TextReader._read_rows
  File "pandas\_libs\parsers.pyx", line 848, in pandas._libs.parsers.TextReader._tokenize_rows
  File "pandas\_libs\parsers.pyx", line 859, in pandas._libs.parsers.TextReader._check_tokenize_status
  File "pandas\_libs\parsers.pyx", line 2025, in pandas._libs.parsers.raise_parser_error
pandas.errors.ParserError: Error tokenizing data. C error: Expected 1 fields in line 6, saw 5
```